text/atlas.go

```diff
@@ -9,9 +9,13 @@ import (
 
 	"github.com/faiface/pixel"
 	"golang.org/x/image/font"
+	"golang.org/x/image/font/basicfont"
 	"golang.org/x/image/math/fixed"
 )
 
+// Atlas7x13 is an Atlas using basicfont.Face7x13 with the ASCII rune set
+var Atlas7x13 *Atlas
```

Comment:
Simple direct assignment will be better and cleaner instead of putting the code into the init function and defining an extra new7x13Atlas function.

Reply:
But ASCII is set in init, that is why I didn't want to set it before init is run.

Reply:
Oh, that's right indeed. In that case, it must be in init, but the new7x13Atlas function is redundant anyway. That one should go.

Reply:
I've removed `new7x13Atlas` in favor of using `NewAtlas` in `init`